docs: Add disable iOS low power mode suggestion (#15110)
immich-app/immich: docs/docs/FAQ.mdx

On iOS (iPhone and iPad), the operating system determines if a particular app can invoke background tasks based on multiple factors, most of which the Immich app has no control over. To increase the likelihood that the background backup task is run, follow the steps below:

- Enable Background App Refresh for Immich in the iOS settings at `Settings > General > Background App Refresh`.

> - Disable low power mode when not needed as this can reduce the amount apps can do in the background.

Reviewer: ```suggestion
- Disable **Low Power Mode** when not needed, as this can prevent apps from running in the background.
```
